### PR TITLE
RHOAIENG-14416 - Implement check for missing tests

### PR DESCRIPTION
**Pull request by jiripetrlik**

### CODE REVIEW

**Review by Srihari1192 — DISMISSED** [review]

**Comment by ChughShilpa** [comment]
This check for missing tests needs to add for kuberay tests also 
@jiripetrlik

File: ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource [hunk 1]
```diff
@@ -212,6 +212,12 @@ Run DistributedWorkloads ODH Test
     ...    env:PIP_INDEX_URL=${PIP_INDEX_URL}
     ...    env:PIP_TRUSTED_HOST=${PIP_TRUSTED_HOST}
     Log To Console    ${result.stdout}
+    Check missing Go test    ${result.stdout}
```
[the remark this answers is not in the document]
Comment (ChughShilpa) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:215]: We observed this issue while running tests from upstream repos like kuberay

**Comment by jiripetrlik** [comment]
@ChughShilpa Added check also for Ray and TrainingOperator tests.

File: ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource (hunk 1 above)
[the remark this answers is not in the document]
Comment (jiripetrlik) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:216]: Understand, but we can also make mistake or typo. So, it is better to check. This check is immediate and takes no time.

[the remark this answers is not in the document]
Comment (ChughShilpa) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:216]: If it doesn't take much time, am fine to keep it

**Review by ChughShilpa — APPROVED** [review]
/lgtm

**Pull request merged**